Lesson3/Serhii/mail.cpp

```diff
@@ -0,0 +1,33 @@
+#include <string>
+#include <iostream>
+
+bool isValid(const std::string & value) {
+    int coefficient = 0;
+    for (const char & c : value) {
+        if (c == ')') {
+            --coefficient;
+        } else if (c == '(') {
+            ++coefficient;
+        }
+        if (0 > coefficient) {
+            return false;
+        }
+    }
+    return !coefficient;    
+}
+
+int main() {
+    std::cout << "Task 1" << std::endl;
```

Comment:
This is too easy for you. Originally, this task includes input with "{}", and "[]". So, please modify your code according to this requirements.
 For example, the program should print true for exp = “[()]{}{[()()]()}” and false for exp = “[(])”